common/src/main/protobuf/validation.proto

```diff
@@ -8,7 +8,13 @@ message ValidationErrorMessage {
     string editName = 2;
     ValidationErrorTypeMessage validationErrorType = 3;
     ValidationErrorEntityMessage validationErrorEntity = 4;
-    map<string, string> fields = 5;
+    repeated string fieldsKey = 5;
+    repeated string fieldsValue = 6;
+}
+
+message ProtoDetailsPair {
```

Comment:
Is this being used? If not, please take it out. Other than this everything else LGTM